src/main/java/io/tja/osrs/shattered_relics_fragment_presets/ShatteredRelicsFragmentPresetsSidebarOverlayPanel.java

```diff
@@ -24,6 +24,7 @@
     private final int SIDEBAR_WIDTH = 120;
     private final int SIDEBAR_RIGHT_MARGIN = 12;
     private final int SIDEBAR_TOP_MARGIN = 4;
+    private final int SIDEBAR_X_ADJUSTMENT = 620;
 
```

Comment:
620 pixels moves the menu to the right of the Shattered Relics menu. The right side has a safer margin in the various scaling and resize modes of OSRS and RuneLite.